src/tests/ftest/dfuse/multi_user.py

```diff
@@ -50,10 +50,18 @@ def test_dfuse_mu_mount(self):
             expect_rc=0)
         ret0 = ret[0]
         print(ret0)
-        assert ret0['exit_status'] == 0
+        self.assertEqual(ret0['exit_status'], 0)
+
+        ret = general_utils.run_pcmd(
+            self.hostlist_clients,
+            'daos container get-attr --path {}/new-cont'.format(root_dir),
+            expect_rc=0)
+        ret0 = ret[0]
+        print(ret0)
+        self.assertEqual(ret0['exit_status'], 0)
```

Comment:
What if `ret[0] == 0` but `ret[1:]` is not?

Reply:
there is only one server/client.  Perhaps an assert "len(ret) == 1" but it wouldn't make the code more readable.

Reply:
Gotcha. If you wanted robustness and readability, you could do something like this
```
from run_utils import run_remote
result = run_remote(self.log, self.hostlist_clients, command)
self.assertTrue(result.passed, 'Failed to do the thing')
```
Where `result.passed` effectively does `len(ret) == 1 and ret[0] == 0`

Reply:
I don't know why I didn't notice before, but we should handle this with existing functions. Like this (includes some other cleanup to align with ftest): https://github.com/daos-stack/daos/commit/219d778015d4a8953c80752dedda9647c029b3e3